crates/egui/src/containers/scroll_area.rs

```diff
@@ -834,11 +835,17 @@ impl Prepared {
                     } else {
                         // Ui is already in view, no need to adjust scroll.
                         0.0
-                    }
+                    };
+                    Some((delta, animation))
```

Comment:
Shouldn't this be

```suggestion
                    Some((delta, target.animation))
```

?

Reply:
Ah yes, must have forgotten that when introducing the ScrollTarget struct